logic/dapp.js

```diff
@@ -79,8 +81,6 @@ DApp.prototype.calculateFee = function() {
  * @todo Add description for the params
  */
 DApp.prototype.verify = function(transaction, sender, cb, tx) {
```

Comment:
What is the reason for not writing function classes? Like:
```js
class DApp {
  constructor(db, logger, schema, network) {
    ...
  }
  verify(transaction, sender, cb, tx) {
    ...
  }
}
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes